xsoar/resource_integration_instance.go

```diff
@@ -702,45 +695,36 @@ func (r resourceIntegrationInstance) ImportState(ctx context.Context, req tfsdk.
 		}
 	}
 
-	//var integrationConfigs map[string]attr.Value
-	integrationConfigs := make(map[string]attr.Value)
+	integrationConfigs := make(map[string]any)
 	if integration["data"] == nil {
-		integrationConfigs = map[string]attr.Value{}
+		integrationConfigs = map[string]any{}
 		log.Println(integrationConfigs)
 	} else {
 		var integrationConfig map[string]interface{}
-		var valueattr attr.Value
 		switch reflect.TypeOf(integration["data"]).Kind() {
 			case reflect.Slice:
 				s := reflect.ValueOf(integration["data"])
 				for i := 0; i < s.Len(); i++ {
 					integrationConfig = s.Index(i).Interface().(map[string]interface{})
 					log.Println(integrationConfig)
-
-					valueconf, ok := integrationConfig["value"].(string)
-					if ok {
-						valueattr = types.String{ Value: valueconf,}
-					} else {
-						valueattr = types.String{ Value: "",}
-					}
-
 					nameconf, ok := integrationConfig["name"].(string)
 					if ok {
-						integrationConfigs[nameconf] = valueattr.(attr.Value)
+						integrationConfigs[nameconf] = integrationConfig["value"]
 					} else {
 						break
 					}
 				}
 		}
 	}
+	integrationConfigsJson, _ := json.Marshal(integrationConfigs)
```

Comment:
I'd add the error to resp.Diagnostics instead of ignoring it. [developer.hashicorp.com/terraform/plugin/framework/diagnostics](https://developer.hashicorp.com/terraform/plugin/framework/diagnostics)